process-document/src/main/kotlin/com/ritense/processdocument/service/CorrelationServiceImpl.kt

```diff
@@ -60,22 +60,22 @@ class CorrelationServiceImpl(
         targetProcessDefinitionKey: String
     ){
         val processDefinitionId = getLatestProcessDefinitionIdByKey(targetProcessDefinitionKey)
-        val correlationResultProcess = correlateWithProcessDefinitionId(message,businessKey, processDefinitionId.id,variables)
+        val correlationResultProcess = correlateWithProcessDefinitionId(message, businessKey, processDefinitionId.id, variables)
         val processName =
             camundaProcessService.findProcessDefinitionById(correlationResultProcess.processDefinitionId).name
         associateDocumentToProcess(correlationResultProcess.processInstanceId, processName, businessKey)
     }
 
     override fun sendCatchEventMessage(message: String): MessageCorrelationResult{
```

Comment:
Isn't the 'send' and 'catch' part of this name a little conflicting? Same goes for the other methods with the same name.